index.html

```diff
@@ -36,5 +36,166 @@ <h1>I'm Ivonne. Glad to see you!</h1>
+                            <p class="socialtext-part about-me-part">
+                                LET'S CONNECT
+                            </p>
+                            <ul class="socialnetwork-icon about-me-part">
+                                <li><a href="#"><img src="./img/icons-social-network/Tertiary/Icons/frame.svg" alt="github logo"></a></li>
+                                <li><a href="#"><img src="./img/icons-social-network/Tertiary/Icons/frame-1.svg" alt="linkedin logo"></a></li>
+                                <li><a href="#"><img src="./img/icons-social-network/Tertiary/Icons/angelist 1.svg" alt="angelist logo"></a></li>
+                                <li><a href="#"><img src="./img/icons-social-network/Tertiary/Icons/frame-2.svg" alt="twitter logo"></a></li>
+                                <li><a href="#"><img src="./img/icons-social-network/Tertiary/Icons/icon-medium.svg" alt="medium logo"></a></li>
+                            </ul>
+                    </div>
+                    <div class="normal-button resume-btn">
+                        <button class="normal-btn"><a href="#">Get my resume</a></button>
+                    </div>
+                    <div class="languages">
+                        <div class="lang-chv">
+                            <div class="title-icon-me-part">
+                                <h4 class="title">Languages</h4>
+                                <img src="./img/icons-social-network/Tertiary/Icons/chv-open.svg" class="chev-open" alt="chv-open">
+                            </div>    
+                            <ul class="languages-icon">
+                                <li><img src="./img/icons-social-network/Tertiary/Icons/JS.png" alt="JavaScript logo"><a href="#"><p class="lang">JavaScript</p></a></li>
+                                <li><img src="./img/icons-social-network/Tertiary/Icons/HTML.png" alt="HTML logo"><a href="#"><p class="lang">HTML</p></a></li>
+                                <li><img src="./img/icons-social-network/Tertiary/Icons/CSS.png" alt="CSS logo"><a href="#"><p class="lang">CSS</p></a></li>
+                            </ul>
+                        </div>
+
+                       <div class="fram-chv">
+                            <div class="title-icon-me-part">
+                                <h4 class="title">Frameworks</h4>
+                                <img src="./img/icons-social-network/Tertiary/Icons/chevron-right.svg" alt="chv-close">
+                            </div>
+                        </div>
+                        <div class="skills-chv">
+                            <div class="title-icon-me-part">
+                                <h4 class="title">Skills</h4>
+                                <img src="./img/icons-social-network/Tertiary/Icons/chevron-right.svg" alt="chv-close">
+                            </div>
+                        </div>
+                    </div>
```

Comment:
- [ ] Great job on your project! Congratulations! I have a friendly suggestion to enhance its structure. In this particular section, you might want to consider using an unordered list `ul` with list items `li` instead of divs for each category like languages, frameworks, and skills. Here's an example of how it could look:

```
<ul>
  <li>
    <hx>Languages</hx>
    <ul>
      <li>JavaScript</li>
      ...
    </ul>
  </li>
  <li>
    <hx>Frameworks</hx>
  </li>
  <li>...</li>
</ul>
```

Remember that it's a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md):
![image](https://github.com/IvonneBenitesRodriguez/microverse_mobile_portfolio/assets/107952724/2cdd450a-b8d6-4773-af6b-946304aff38a)

Reply:
👋🏾 @taldr27 
Thank you for your advice, so I have a question , and how do I place a chevron icon? it is at the same level of the languages categories which is why i created a div to use display flex and place it, but according your suggestion 
can you please let me know how can I place the chevron icon at the same level of the languages category?
Also, it happens the same with the skill and framework category, we need to place a chevron icon svg ?
thank you it will help me a lot,

Reply:
Hello, @IvonneBenitesRodriguez, you can apply display flex to the li element too without any problems.

Reply:
Dear @taldr27 
Thank you so much for the suggestion, 🙏🏿
I modified the changes you suggested, thank you again it helped me to learn more about nested li tags ⭐️
Best regards,